client/styles/main.scss

```diff
@@ -34,6 +34,7 @@
 @import 'components/toast';
 @import 'components/timer';
 @import 'components/force-authentication';
+@import 'components/form-container';
```

Comment:
The only error I found was that you need to remove the following lines from this file:
```scss
@import 'components/signup';
@import 'components/login';
```
since you deleted those files! 😄